test-apps/remix-vite/icons/test.svg

```diff
@@ -1,3 +1,3 @@
-<svg>
+<svg xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
without this, svgo would yell at me
```
SvgoParserError: <input>:2:24: Unbound namespace prefix: "xlink"
```

Reply:
I'll need to take a deeper dive into this PR before I merge it in, so what will svgo do by default here when ran alongside the plugin?

Reply:
With this error? Crash the plugin and build sadly :/

Reply:
Sorry, maybe leaving a comment here specifically was a bad idea, this was more of a general question of what will happen when I merge this PR?

Reply:
Svgo is an svg minification tool. This implementation will optimize each svg file individually right before it's being included in the spritesheet as a symbol.

Users can opt-out of Svgo or customize its config by passing the options through the plugin config. I don't know if Svgo can pick up the default `svgo.config.js`, will have to check.

Reply:
I think this change is good because using `xlink:href` inside an `<svg>` element that doesn't have `xmlns:xlink` is probably invalid, that's why SVGO was complaining.